eclass/qt5-build.eclass

```diff
@@ -101,7 +101,16 @@ fi
 : ${QT5_BUILD_DIR:=${S}_build}
 
 LICENSE="|| ( GPL-2 GPL-3 LGPL-3 ) FDL-1.3"
-SLOT=5/$(ver_cut 1-2)
+
+case ${PN} in
+	assistant|linguist|qdbus|qdbusviewer)
```

Comment:
pixeltool should be here, if not dead in qt6. If dead, it should be in the next category (SLOT=5).

Reply:
actually, I think it should be SLOT=0 either way (and symlinked since in Qt6 it's treated as a user-facing app)